docs/src/building-artifacts.md

```diff
@@ -6,6 +6,38 @@ For Ray.jl releases we provide pre-built binary artifacts to allow for [easy ins
 
 Follow [the instructions](https://github.com/beacon-biosignals/ray/blob/beacon-main/python/README-building-wheels.md) outlined in the Beacon fork of Ray.
 
-### JLL Artifacts
+### Artifacts
 
-The `ray_julia_jll` artifacts are hosted in [GitHub releases](https://github.com/beacon-biosignals/Ray.jl/releases) and will be downloaded automatically for any supported platform. To rebuild the artifacts, you can follow the [build instructions](https://github.com/beacon-biosignals/Ray.jl/blob/main/build/README.md).
+The `ray_julia` artifacts are hosted via [GitHub releases](https://github.com/beacon-biosignals/Ray.jl/releases) and will be downloaded automatically for any supported platform. To upate the artifacts, perform the following steps:
+
+1. Update the Ray.jl version in the `Project.toml`
+
+2. Navigate to the `build` directory
+
+3. Run the `build_tarballs.jl` script to build the tarball for the host platform for the Julia version used. Builds are required for `Linux x86_64` and `macOS aarch64` on Julia `v1.8` and `v1.9`. It is advised you run this within the python virtual environment associated with the Ray.jl package to avoid unnecessary Bazel rebuilds.  Note: re-running this script _will overwrite_ an existing tarball for this version of Ray.jl.
+
+   ```sh
+   source ../venv/bin/activate
+   julia --project -e 'using Pkg; Pkg.instantiate()'
+   julia --project build_tarballs.jl
+   ```
+
+4. Run the `upload_tarballs.jl` script to publish the tarball(s) as asset(s) of a GitHub pre-release, which requires a `GITHUB_TOKEN` environment variable. Note: you can run this script after building each tarball, or after you've built them all. Rerunning it will only upload new tarballs and skip any that have already been published.
+
+   ```sh
+   read -s GITHUB_TOKEN
+   export GITHUB_TOKEN
+   julia --project upload_tarballs.jl
+   ```
+
+5. Once all assets are uploaded, checkout a new branch (based off of `main`) and run the `bind_artifacts.jl`. This will modify the local `Artifacts.toml` with the artifact(s) associated with the Ray.jl version specified in the Project.toml. After running this you should commit and push the changes to a new PR.
+
+   ```sh
+   julia --project bind_artifacts.jl
+   ```
+   <!-- TODO: The CI workflows for this PR can run successfully and the referenced artifacts will be accessible. -->
```

Comment:
can remove this now that we have https://github.com/beacon-biosignals/Ray.jl/pull/152
```suggestion
```